src/server/tasks/queue-autosend-initials.ts

```diff
@@ -89,7 +89,8 @@ const queueAutoSendInitials: Task = async (payload: Payload, helpers) => {
           id::text as key,
           null as queue_name,
           1 as max_attempts,
-          now() + ((n / $2::float) * interval '1 second') as run_at
+          now() + ((n / $2::float) * interval '1 second') as run_at,
+          1 as priority
```

Comment:
[Handling delivery reports](https://github.com/politics-rewired/Spoke/blob/main/migrations/20200804190149_deprioritize-handle-delivery-reports.js) is currently at 5 so can only go 1-4 for autosending to keep this behavior we want. I'll move to 4 if that sounds good? And I'll also comment on the 2 jobs the value was decided on relative to, to avoid the detective work 😅